lib/node/actions/http_client.dart

```diff
@@ -5,10 +5,8 @@
 import 'dart:js_interop';
 
 @JS()
-inline class HttpClient {
-  final JSObject obj;
-
-  external HttpClient(
+extension type HttpClient (JSObject obj) {
+  external HttpClient.fromAgent(
```

Comment:
Up to you on how you want to write these, but I've been writing the primary constructor as `HttpClient._(JSObject obj)` when we only ever use the external constructor so that we don't need a named external constructor.

Reply:
Done